package.json

```diff
@@ -18,9 +17,9 @@
     "@tailwindcss/forms": "^0.5.3",
     "@tailwindcss/typography": "^0.5.8",
     "autoprefixer": "^10.4.14",
-    "cable_ready": "^5.0.0",
+    "cable_ready": "5.0.0-pre8",
```

Comment:
@gazayas, this looks like it would downgrade `cable_ready` from `^5.0.0` to `5.0.0-pre8`. Can you fix that and make sure that everything is still good?

Looks like `esbuild` a few lines down would also lose a `^`, which we may not want.

Reply:
Done!